pkg/agent/openflow/client.go

```diff
@@ -1243,7 +1271,12 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8, liveTraffic, droppedO
 
 func (c *client) UninstallTraceflowFlows(dataplaneTag uint8) error {
 	cacheKey := fmt.Sprintf("%x", dataplaneTag)
-	return c.deleteFlows(c.featureTraceflow.cachedFlows, cacheKey)
+	return c.deleteFlows(c.featurePacketSampling.cachedFlows, cacheKey)
```

Comment:
Why is this line changed?

Reply:
it's mistake. Will make sure wont happen again.